_backend/Lib/L10n.php

```diff
@@ -99,21 +99,29 @@ public static function languageDirectory($lang)
     }
 
     protected $available_langs = array(
-        'en' => 'English',
```

Comment:
English must be at the top of the array so it always appears in a separate section at the top of the list.